vimeo-networking/src/main/java/com/vimeo/networking/model/InteractionCollection.java

```diff
@@ -83,6 +83,14 @@
     @SerializedName("moderate_videos")
     private Interaction mModerateVideos;
 
+    @Nullable
+    @SerializedName("add_logos")
+    private Interaction mAddLogos;
+
+    @Nullable
+    @SerializedName("add_videos")
+    private Interaction mAddVideos;
```

Comment:
The uri returned from the API for adding videos to an album is `/users/{user_id}/albums/{album_id}/videos/{video_id}`. The API requires appending the video id to the uri. Providing video id as a query param or part of the request body will not work. There are two options that I think that could be taken:

1. We could add documentation to the `getAddVideos` method to indicate you have to mutate the uri by adding the video id to it.

2. An` addVideosToAlbum` method could be created in the SDK that mutates the uri by appending the video id for the client.

Option 2 is optional. But, I would recommend adding a doc to the `getAddVideos` method.

Reply:
Great suggestion, @msya. I chose option #1. Back to you.